workspaces/linguist/.changeset/orange-cups-switch.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage-community/plugin-catalog-backend-module-linguist-tags-processor': minor
+'@backstage-community/plugin-linguist-backend': minor
+---
+
+**BREAKING** This change removes the deprecated `LinguistTagsProcessor` from `@backstage-community/plugin-linguist-backend`. It also removes the export of `LinguistTagsProcessor` from `@backstage-community/plugin-catalog-backend-module-linguist-tags-processor`. Please install this processor using [the New Backend System setup](https://github.com/backstage/community-plugins/tree/main/workspaces/linguist/plugins/catalog-backend-module-linguist-tags-processor#setup), which is now the default.
```

Comment:
Should breaking changes be allowed in a minor update?

Reply:
I'm following the guidance from the Core Framework for my plugins: https://backstage.io/docs/overview/versioning-policy#package-versioning-policy

As these are not `1.0` yet this would be allowed.

Reply:
Hmm, I thought this was documented a bit better but this is the table I've used in the past: https://github.com/backstage/backstage/blob/master/REVIEWING.md#reviewing-changeset-bump-levels

Reply:
ah, fair point. In "incubating" content <1.0 definitely allowed. 

Sadly this PR doesn't include any mention of the actual version... so I'd have to go look at the current package.json I suppose. 

+1, though I have no idea what the code does - did not test, just skimmed for things like version # bumps 😁

Reply:
The changeset bot can help with that: https://github.com/backstage/community-plugins/pull/1199#issuecomment-2339391021, it lists the current version